images/build/cli.Dockerfile

```diff
@@ -0,0 +1,123 @@
+FROM buildpack-deps:stretch
```

Comment:
Is the purpose of this image to use Oryx's build script generator alone so that it can generate a dockerfile by looking at a repo? If yes, we need not install .NET Core SDK at all since we are doing a self-contained deployment of the build script generator.

Reply:
I mean you can use a multi-stage approach where you install 2.2 SDK in an image, publish Oryx app as standalone and then copy back the content of this stage to the final image (as we do in our other build images)